3기/HyeonkyeongSeo/2주차/css/style.css

```
@@ -0,0 +1,510 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+@font-face {
+    font-family: 'Pretendard-Light';
+    src: url('https://cdn.jsdelivr.net/gh/Project-Noonnu/noonfonts_2107@1.1/Pretendard-Light.woff') format('woff');
+    font-weight: 400;
+    font-style: normal;
+}
+
+
+*, ::before, ::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+    font-family: 'Inter', 'Pretendard-Light', sans-serif;
+  font-size: 14px;
+    color: #2D3436;
+}
+
+
+html, body{
+  line-height: 160%;
+}
+
+a{
+    text-decoration: none;
+    color: inherit;
+}
+
+li{
+    list-style: none;
+}
+
+#l_wrapper{
+    width: 100%;
+    max-width: 100%;
+    height: 100%;
+    min-height: 100vh;
+    display: flex;
+    flex-flow: column;
+    justify-content: space-between;
+    background-color: rgba(217,217,217,0.3);
+}
+
+#wrap{
+    max-width: 1200px;
+    min-width: 320px;
+    width: 90%;
+    margin: 0 auto;  
+    display: flex;
+    position: relative;
+}
```

html 쪽에서도 피드백을 드렸는데, 공통으로 사용되는 콘텐츠 너비를 담당하는 스타일이라면,

```suggestion
.container {
    max-width: 1200px;
    min-width: 320px;
    width: 100%;
    margin: 0 auto; 
}
```

이렇게 제어해주는게 좋아요.

그리고 `display: flex`, `position: relative`를 제거한 이유는,
자식 요소들에게 영향을 주는 스타일은, 필요한 곳에서 추가로 선언해주는게 좋아요.

모든 요소에 공통으로 들어가게 된다면 나중에 오히려 저 속성을 제거하기 위한 속성을 추가로 넣는 모순이 발생할 수 있어요!

---

그리고 width가 90%로 되어있어서 보니까 좌/우 여백이 필요해서 그렇게 처리하신 것 같아요!
이럴 때는 아래처럼 처리하면 좋아요

```suggestion
.container {
    margin: 0 auto;
    padding: 0 32px; /* 테블릿 구간의 좌/우 여백 */
    max-width: calc(1200px + 32px + 32px); /* 테블릿 구간의 좌/우 여백을 더해서 실제 콘텐츠 영역은 1200px을 가져가게 함 */
    min-width: 320px;
    width: 100%;
}
```

이렇게 하면, 브라우저 크기가 조절되어도 테블릿 구간의 좌/우 여백으로 인해서 콘텐츠가 브라우저에 딱 붙어버리는 현상을 예방할 수 있어요

https://user-images.githubusercontent.com/32983425/216756728-a1897677-37cd-47e5-af47-c85db83fdec3.mov

이런식으로요!